clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionManager.java

```diff
@@ -1532,9 +1525,10 @@ String coordinatorKey() {
         }
 
         @Override
-        public void handleResponse(AbstractResponse response) {
+        public void handleResponse(AbstractResponse response, short requestVersion) {
             FindCoordinatorResponse findCoordinatorResponse = (FindCoordinatorResponse) response;
             CoordinatorType coordinatorType = CoordinatorType.forId(builder.data().keyType());
+            boolean batchFindCoordinator = requestVersion >= FindCoordinatorRequest.MIN_BATCHED_VERSION;
```

Comment:
The above suggestion would also us to avoid having to pass the `requestVersion` down here.